src/sarif_converter.ts

```diff
@@ -149,12 +159,13 @@ const getResults = (suggestions: ISarifSuggestions) => {
               },
             },
           });
-          // i += 1;
+          i += 1;
         }
       }
     } else {
       codeThreadFlows.push({
         location: {
+          id: 0,
```

Comment:
`question`: Why is id 0 here? Is this arbitrary value required somewhere else? How shall we make sure consumer is aware of this setup?

Reply:
this is the id of if there is no code flow so there is only 1 item giving it an id of 0